build/Linux+BSD/portable/copy-libs

```diff
@@ -116,8 +239,6 @@ building_on_CentOS_6() {
   copyLib libQt5Concurrent.so.5
   copyLib libQt5DBus.so.5
   copyLib libQt5Designer.so.5
-  copyLib libQt5Location.so.5
-  copyLib libQt5QuickParticles.so.5
```

Comment:
Looks good! I'll try to write an "ignore" function for check-depends so it doesn't report them as missing.

Any directories in qt5/qml that are related to those libraries can be removed too. Simply add the name of the directory to this [exclude list](https://github.com/musescore/MuseScore/pull/2491/files#diff-648855eea2924892f388f6690d940de6R105).